code/modules/surgery/implant.dm

```diff
@@ -175,6 +175,12 @@
 		to_chat(user, "<span class='warning'>Central command would kill you if you implanted the disk into someone.</span>")
 		return 0//fail
 
+	if(istype(tool, /obj/item/weapon/storage/box))
+		for(var/obj/item/I in tool.contents)
```

Comment:
Use `GetAllContents(tool)` instead. Or someone will put a wallet inside a box or something and put it in.